keyboards/czmao/ds17/readme.md

```diff
@@ -0,0 +1,6 @@
+# mao ds 17
```

Comment:
Please re-add full readme, and make sure to include a link to where the hardware can be acquired.

Reply:
updated